apps/backend/src/modules/auth/libs/enums/enums.ts

```diff
@@ -1 +1 @@
-export { AuthApiPath } from "shared";
+export { AuthApiPath, HTTPCode, UserValidationMessage } from "shared";
```

Comment:
HTTPCode should be exported from http module. UserValidationMessage should be exported from user module